### PR TITLE
Add PR review value to README.md

### PR DESCRIPTION
**Pull request by nbolton**
Proposed additional value:

> - All pull requests are reviewed and approved (apart from minor doc changes)

We believe a project that consistently lands PRs without review can give the impression of being driven by a single individual, and bypassing review processes risks setting an unbalanced precedent for the team.

Our philosophy I that decisions should be made collectively, and that applies to deciding what code to land in `master`. We already have required PR review/approval in our branch protection rules, of course, but adding this to the `README` makes it clear to our community (especially newcomers) where we stand.

I added one caveat to this value, which I'm happy to forgo if it's undesirable:

> (apart from minor doc changes)

By this, I mean things like minor typo fixes, broken link/image fixes, etc to files like the README.md, wiki, etc.

Since this PR is a major change (amending our project values), I think it deserves a review.

### CODE REVIEW

**Comment by nbolton** [comment]
Just noticed the commit message, will fix.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ install `deskflow` (see: [installing packages](#how-to-install-packages)).
 - Privacy by default (e.g. update check is off by default)
 - Nothing customer-related (this is all moved downstream to Synergy)
 - Decisions are discussed and documented publicly with majority rule
+- All pull requests are reviewed and approved (apart from minor doc changes)
```
Comment (nbolton) [inline README.md:54]: Or maybe...
```
- All pull requests require review and approval (apart from minor doc changes)
```

Comment (sithlord48) [reply README.md:54]: Imho this while good to have does not belong in this document.
Really this kinda of info (about the dev /  review process) should go into the `CONTRIBUTING` file were we can point out what is expected of contributors.

Comment (nbolton) [reply README.md:54]: Do you think the whole values section should go in `CONTRIBUTING` there or just this one line? Like you said, the README got rambly and we need to fix that.

**Review by sithlord48 — CHANGES_REQUESTED** [review]
lets do this more inline with #7665

**Comment by nbolton** [comment]
> lets do this more inline with #7665

Cool, I'll close. Added a note to that issue.

**Pull request closed without merging**